packages/custom-url/src/Infrastructure/Sulu/EventSubscriber/CacheInvalidationSubscriber.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\CustomUrl\Infrastructure\Sulu\EventSubscriber;
```

Comment:
```suggestion
namespace Sulu\CustomUrl\Infrastructure\Sulu\HttpCache;
```

there is no event subscriber bundle this is connection point to the http cache bundle